src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
```

Comment:
There is an unnecessary empty line at the beginning of the 'main' method implementation.